build.cmd

```diff
@@ -23,7 +23,7 @@ set "__LogsDir=%__RootBinDir%\Logs"
 set __MSBCleanBuildArgs=
 set __SkipTestBuild=
 set __ToolchainMilestone=testing
-set __DotNetCliPath=
+set "__DotNetCliPath=%__ProjectDir%\Tools\dotnetcli"
 
```

Comment:
Ah, the `DotnetCLIVersion.txt`. The version that works for us may not work for msbuild, but we are nearing CLI RC2 and RTM, so things should stabilize...